boards/riscv/tlsr9118bdk40d/tlsr9118bdk40d-common.dtsi

```diff
@@ -113,3 +125,9 @@
 	pinctrl-0 = <&i2c_scl_default &i2c_sda_default>;
 	pinctrl-names = "default";
 };
+
+&pwm0 {
+	status = "okay";
+	clock-frequency = <1000000>;
```

Comment:
Where do we set this frequency?
Is it some constant value? If yes set it in the yaml.

Reply:
This value from FREERTOS core, but I've just found that we can use API to get this vakue. I will rework this part

Reply:
as I see the situation is the same...
set It to const, othewise changing this value will fail the driver.

Reply:
I don't use this value anymore, I just forgot to remove it